### ISSUE TITLE
Tongue hyperpigmentation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11691]
# Tongue hyperpigmentation

## Proposed definition

Increased pigmentation, either focal or generalized, of the surface of the tongue.

## Parent term

- `HP:0010284 Intra-oral hyperpigmentation`

## Synonyms

- Hyperpigmentation of the tongue
- Lingual hyperpigmentation

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617
@evanc1648

**Comment by pnrobinson** [comment]
@evanc1648

Adding new term: 
- ID: HP:6001529
- label: Tongue hyperpigmentation
- parents: [Intra-oral hyperpigmentation (HP:0010284)](https://hpo.jax.org/app/browse/term/HP:0010284)
- pmid: 
- def: Increased pigmentation, either focal or generalized, of the surface of the tongue.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11691
- synonyms: Lingual hyperpigmentation(NONE)|Hyperpigmentation of the tongue(NONE)
